src/extensions/sonarcloud/overview.md

```diff
@@ -8,14 +8,14 @@ This Azure DevOps extension provides build tasks that you can add in your build
 
 To get started in a few minutes, you can:
 
-- Follow this dedicated [Microsoft Lab][getstarted]
-- Follow the SonarQube Cloud walkthrough in this Microsoft Learning Module: [Scan code for vulnerabilities in Azure Pipelines][msft_learn].
+- Check out the official [SonarQube Cloud documentation][documentation].
+- Follow this dedicated [Azure DevOps Labs][getstarted].
 - Benefit from embedded templates for common analyses:
   - **Classic** build pipelines templates integrated while creating a new pipeline
   - **YAML** build pipeline templates available in our Sourcecode: https://github.com/SonarSource/sonar-scanner-azdo/tree/master/its/fixtures
-- Benefit from a bunch of sample projects and pipelines (used in our automated tests). See the projects (.NET Framework, .NET Core, Gradle, Maven, Scanner CLI) [on GitHub](./its/fixtures) and the corresponding analyzed projects in [the SonarQube Cloud organization](https://sonarcloud.io/organizations/sonarsource-sonar-scanner-azdo-it/projects?sort=analysis_date).
```

Comment:
This returns a 404. I could not find the example projects analyzed on SonarQube Cloud.